LiskHQ/lisk-nano
# Setup auto update - Closes #196

### What was the problem?
Users stay stayed on older versions without knowing there is a new one.

### How did I fix it?
By setting up [electron-updater](https://www.electron.build/auto-update)

### How to test it?
- decrease version in `app/package.json` to `1.2.0`
- build nano with `npm run pack` (On mac) or `npm run pack:win` (On Windows)
- install the built package from 'dist' folder
- open lisk nano
- wait for couple seconds
- there will be an alert like this: <img width="875" alt="screen shot 2017-11-17 at 14 30 13" src="https://user-images.githubusercontent.com/1254342/32949800-6450de9a-cba4-11e7-958d-ccd610e3ee33.png">
- click "Update now"
- wait for a minute or two for the download (you can use that time to check version in about dialog or log in and check that "encrypt message" feature is present)
- there will be another alert when the download finishes, confirm it.
- the app restarts
- check version in about dialog or log in and check that "encrypt message" feature is no longer present

### Review checklist
- [ ] The PR solves #196
- [ ] Works on Mac and Windows (Linux will be handled separately https://github.com/LiskHQ/lisk-nano/issues/991)
